platforms/gke/base/use-cases/federated-learning/README.md

```diff
@@ -1 +1,39 @@
 # Federated learning on Google Cloud
+
+## Configure the Federated learning reference architecture
+
+1. Enable Binary Authorization. Configure the following variables in
+   `platforms/gke/base/_shared_config/cluster.auto.tfvars`:
+
+   ```hcl
+   cluster_binary_authorization_evaluation_mode = "PROJECT_SINGLETON_POLICY_ENFORCE"
+   ```
+
+1. If needed, enable GKE Confidential Nodes. Configure the following variables
+   in `platforms/gke/base/_shared_config/cluster.auto.tfvars`:
+
+   ```hcl
+   cluster_confidential_nodes_enabled = true
+   ```
+
+## Deploy the Federated learning reference architecture
+
+1. Provision the base platform by following the
+   [Core GKE Accelerated Platform guide](/platforms/gke/base/core/README.md).
+
+1. Provision the Federated Learning reference architecture:
+
+   ```sh
+   "${ACP_PLATFORM_BASE_DIR}/use-cases/federated-learning/deploy.sh"
+   ```
+
+## Teardown the Federated Learning reference architecture
+
+1. Teardown the Federated Learning reference architecture:
+
+   ```sh
+   "${ACP_PLATFORM_BASE_DIR}/use-cases/federated-learning/teardown.sh"
+   ```
+
+1. Teardown the base platform by following the
+   [Core GKE Accelerated Platform guide](/platforms/gke/base/core/README.md#teardown).
```

Comment:
I would incorporate all of this into your deployment script.

Reply:
Makes sense. This is a temporary placeholder that I'm using as a reminder. Keeping this open so we don't forget about this

Reply:
Done